### PR TITLE
style: Investigate pylint for PEP8

### PR DESCRIPTION
**Pull request by ross-p-smith**
## Purpose
Wanted to investigate the benefit of using pylint which seems to give us better vscode integration.

This PR is more for discussion as it relates #864

### CODE REVIEW

File: code/create_app.py [hunk 1]
```diff
@@ -180,45 +187,51 @@ def conversation_with_data(request: Request, env_helper: EnvHelper):
         }
 
         return response_obj
-    else:
-        return Response(
-            stream_with_data(response),
-            mimetype="application/json-lines",
-        )
+
```
Comment (ross-p-smith) [inline code/create_app.py:190]: https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/no-else-return.html

File: code/create_app.py [hunk 2]
```diff
@@ -271,10 +285,8 @@ def conversation_without_data(request: Request, env_helper: EnvHelper):
             ],
         }
         return jsonify(response_obj), 200
-    else:
-        return Response(
-            stream_without_data(response), mimetype="application/json-lines"
-        )
+
```
Comment (ross-p-smith) [inline code/create_app.py:288]: https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/no-else-return.html

File: code/app.py [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+# https://github.com/Azure/azure-sdk-for-python/issues/9422
 logging.getLogger("azure").setLevel(os.environ.get("LOGLEVEL_AZURE", "WARN").upper())
 # We cannot use EnvHelper here as Application Insights should be configured first
 # for instrumentation to work correctly
 if os.getenv("APPLICATIONINSIGHTS_ENABLED", "false").lower() == "true":
     configure_azure_monitor()
     HTTPXClientInstrumentor().instrument()  # httpx is used by openai
 
+# pylint: disable=wrong-import-position
 from create_app import create_app  # noqa: E402
```
Comment (gaurarpit) [inline code/app.py:22]: just realized, shouldn't this move to the top of the file ? ( we did that story a fews day back, right ? )

File: code/create_app.py (hunk 1 above)
Comment (gaurarpit) [reply code/create_app.py:190]: ahaa.. this is so nice. My pet peeve is when else block literally has only 1 line that returns.

**Review by gaurarpit — APPROVED** [review]

File: code/app.py (hunk 3 above)
Comment (ross-p-smith) [reply code/app.py:22]: This is for this 1 particular line - this is so we setup logging before we create the app, but also allows us to test the creat_app code without all the logger infrastructure

**Pull request merged**